emails/tests/views_tests.py

```diff
@@ -137,31 +139,110 @@ def create_email_from_notification(
     return email.as_bytes()
 
 
-def assert_email_equals(output_email: str, name: str) -> None:
+def create_notification_from_email(email_text: str) -> AWS_SNSMessageJSON:
     """
-    Assert the output equals the expected email, after replacements.
+    Create an SNS notification from a raw serialized email.
 
-    This function replaces MIME boundary strings, which change each test run.  The
-    replacement is "==[BOUNDARY#]==", where "#" is the order the string appears in the
-    email.
+    The notification will have the headers from the email, a passing receipt,
+    and other mocked items. The email will be included in the notification
+    body, not loaded from (mock) S3.
+    """
+    email = message_from_string(email_text, policy=policy.default)
+    topic_arn = "arn:aws:sns:us-east-1:168781634622:ses-inbound-grelay"
+    sns_message = {
+        "notificationType": "Received",
+        "mail": {
+            "timestamp": email["Date"].datetime.isoformat(),
+            "source": email["From"].addresses[0].addr_spec,
+            "messageId": email["Message-ID"],
+            "destination": [addr.addr_spec for addr in email["To"].addresses],
+            "headersTruncated": False,
+            "headers": [{"name": _h, "value": _v} for _h, _v in email.items()],
+            "commonHeaders": {
+                "from": [str(addr) for addr in email["From"].addresses],
+                "date": email["Date"],
+                "to": [str(addr) for addr in email["To"].addresses],
+                "messageId": email["Message-ID"],
+                "subject": email["Subject"],
+            },
+        },
+        "receipt": {
+            "timestamp": (email["Date"].datetime + timedelta(seconds=1)).isoformat(),
+            "processingTimeMillis": 1001,
+            "recipients": [addr.addr_spec for addr in email["To"].addresses],
+            "spamVerdict": {"status": "PASS"},
```

Comment:
question (non-blocking): does this need to be `"status": "FAIL"` for the russian spam fixture?

Reply:
Maybe, but we're not testing spam handling here. That's `spamVerdict_FAIL_email_sns_body.json`. If we can get a legitimate email with a lot of non-ASCII content, we should replace this fixture to remove confusion